src/pages/recruit/RecruitCard.tsx

```diff
@@ -0,0 +1,122 @@
+import { Button } from '@/components/common';
+import instagram from '@assets/styles/images/features/layout/footer/instagram_Icon.svg';
```

Comment:
```suggestion
import instagram from '@assets/icons/social/instagram.svg';
```

이렇게 변경되어야할 것 같네요.. ㅎㅎ

path alias를 굳이 써야하는 이유가 있을까요?
지금처럼 소스코드 추적도 잘 안 되는 상황이다보니..